src/core/handleErrorFactory.js

```diff
@@ -12,9 +12,12 @@ governing permissions and limitations under the License.
 
 import { toError } from "../utils";
 
+const suppressionNote =
+  "Note: Errors can be suppressed via the errorsEnabled configuration option.";
+
 export default ({ instanceNamespace, getErrorsEnabled, logger }) => error => {
   const err = toError(error);
-  err.message = `[${instanceNamespace}] ${err.message}`;
+  err.message = `[${instanceNamespace}] ${err.message}\n${suppressionNote}`;
   if (getErrorsEnabled()) {
     throw err;
   } else {
```

Comment:
Assuming logging is enabled, the suppressionNote will be logged even when errors are not enabled.

Reply:
That's a good point. I should only add the note if errorsEnabled is false. I'll make the change.

Reply:
I mean....true.